components/typography/test/typography.vdiff.js

```diff
@@ -0,0 +1,54 @@
+import '../typography.js';
```

Comment:
So including this is a little bit conflict-y with [what vdiff is already including](https://github.com/BrightspaceUI/testing/blob/main/src/server/wtr-config.js#L71). It's _probably_ fine, but we'll need to be careful to keep them in sync if we ever publish new versions of those assets to the CDN.

Reply:
Yeah I can't really think of a way around it to still test the `typography` code, unless we isolated the typography tests into their own job and created a custom `testRunnerHtml` for them?

Reply:
Yeah doesn't seem worth the trouble since this is working.